id-repository/id-repository-identity-service/src/test/java/io/mosip/idrepository/identity/test/service/impl/IdRepoServiceTest.java

```diff
@@ -1144,26 +1201,13 @@ public void testEncryptDecryptDocumentsNoData() throws Throwable {
 		}
 	}
 
-	@Test(expected = IdRepoAppException.class)
-	public void testConvertToFMR() throws Throwable {
-		try {
-			MockEnvironment mockEnv = new MockEnvironment();
-			mockEnv.merge((ConfigurableEnvironment) env);
-			mockEnv.setProperty("mosip.fingerprint.fmr.enabled", "true");
-			ReflectionTestUtils.setField(service, "env", mockEnv);
-			when(cbeffUtil.updateXML(Mockito.any(), Mockito.any())).thenThrow(new NullPointerException());
-			ReflectionTestUtils.invokeMethod(service, "convertToFMR", "123", "123");
-		} catch (UndeclaredThrowableException e) {
-			throw e.getCause();
-		}
-	}
-
```

Comment:
why is this test removed ?

Reply:
That method "convertToFMR" was removed. Earlier this whole test class was ignored. When I removed the ignore found this test no more valid as the method is not present.